src/config/index.ts

```diff
@@ -31,7 +31,7 @@ export type HierarchyConstraint = {
   property_name: string;
   type: string;
   required: boolean;
-  parameter? : string | string[];
+  parameter? : string | string[] | object;
```

Comment:
should there be a change to Uganda's `config.json` file? it uses gender which now no longer exists

Reply:
Yes